rpc/api.proto

```diff
@@ -64,6 +64,23 @@ service SeedService {
 	rpc DecodeSeed (DecodeSeedRequest) returns (DecodeSeedResponse);
 }
 
+service TicketBuyerService {
+    rpc StartTicketPurchase (StartTicketPurchaseRequest) returns (StartTicketPurchaseResponse);
+    rpc StopTicketPurchase (StopTicketPurchaseRequest) returns (StopTicketPurchaseResponse);
```

Comment:
Please define these as part of the WalletLoaderService since this requires touching the loader and to keep it consistent with starting the wallet.